etc/ftest_flow.txt

```diff
@@ -0,0 +1,86 @@
+/*// This is a simple mimic of the file flow
```

Comment:
Can we delete this file? Why does it exist?

Reply:
Deleted.